src/components/Header/style.scss

```diff
@@ -21,7 +21,8 @@ $HEADER_MAX_SIZE: 54 !default;
     margin: 0 0 0 auto;
   }
 
-  .powerIcon {
+  .powerIcon, .settingsIcon {
+    margin-left: 20px;
```

Comment:
It's better to use something more responsive like rems, because pixels are always the same, no matter on which resolution